helm_deploy/court-hearing-event-receiver/templates/deployment.yaml

```diff
@@ -0,0 +1,67 @@
+apiVersion: apps/v1
```

Comment:
My understanding is that with the `generic-service` chart imported we shouldn't need the majority of these templates as they're all provided [here](https://github.com/ministryofjustice/hmpps-helm-charts/tree/main/charts/generic-service/templates)